src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -132,8 +134,11 @@ function BankAccountPlaidStep(props) {
 BankAccountPlaidStep.propTypes = propTypes;
 BankAccountPlaidStep.defaultProps = defaultProps;
 BankAccountPlaidStep.displayName = 'BankAccountPlaidStep';
-export default withOnyx({
-    plaidData: {
-        key: ONYXKEYS.PLAID_DATA,
-    },
-})(BankAccountPlaidStep);
+export default compose(
```

Comment:
Let's remove the `useLocalize` hook and use the `translate` from the `props`.

Reply:
Makes sense. updated